ZAPDTR/ZAPD/ZFile.cpp

```diff
@@ -800,9 +800,12 @@ void ZFile::GenerateSourceHeaderFiles()
 {
 	OutputFormatter formatter;
 
+	formatter.Write("#pragma once\n");
+	bool first = true;
```

Comment:
We should maybe give this a more descriptive name. So we're trying to remember if we've already generated the header for at least one resource?

Reply:
removed "first" variable entirely.